applications/OptimizationApplication/python_scripts/utilities/communicators/response_function_communicator.py

```diff
@@ -0,0 +1,199 @@
+import KratosMultiphysics as Kratos
```

Comment:
please check my point in optimization_component_communicator. We can discuss further

Reply:
Moved all the common methods to `StandardizationUtilities`, and removed all the `Communicator` classes.